data/aac-missing-native-codec.yml

```diff
@@ -0,0 +1,59 @@
+title: AAC audio decoding fails if there is no native codec
+severity: low
+user_base_impact: small
+
+tags:
+  - media
+
+notes: |
+  Due to patent issues, Firefox does not ship an AAC codec. Instead, it relies
+  on the operating system to provide one. In certain circumstances, it's
+  possible that the OS doesn't provide a codec, thus breaking AAC playback.
+
+symptoms:
+  - Issues with Audio or Video playback
+  - Error messages claiming that a codec is not supported
+
+references:
+  breakage:
+    - url: https://webcompat.com/issues/9557
+      site: http://positivr.fr/bucheron-cabestan-medieval-bois/
+      platform:
+        - linux
+      resolution: site_changed
+      impact: feature_broken
+      affects_users: few
```

Comment:
To be clear: this is "few" as in "only a small subset of linux users", right? Not "few" as in "there are few linux users"? (because we already account for the latter).

Reply:
Yeah. This only affects Linux setups that don't have `gstreamer1.0-libav` installed, which shouldn't be too common. On Ubuntu, the codec is installed via `ubuntu-restricted-extras`. The Snap works as well.

We've seen a bunch of reports from Mint users, and one Gentoo user. But it's definitely not "all" Linux users, and I don't even think it's "some" Linux users. The vast majority should be fine as far as I can tell, but maybe @wisniewskit knows more.

Reply:
Dennis basically summed up the situation, it's just down to some Linux users not having the expected codec package installed. It doesn't affect all Linux users, it's dependent on whether the specific distribution defaults to enabling the codecs or not.